redbot/cogs/trivia/data/lists/sports.yaml

```diff
@@ -31,7 +31,7 @@ How many times has a Super Bowl went into overtime?:
 How many world records did swimmer Mark Spitz set when he won seven gold medals at the 1972 Olympics?:
 - 7
 - seven
-How much money is a US Olympic gold medalist awarded?:
+How much money is a US Olympic gold medallist awarded?:
 - $25,000
```

Comment:
If you want to correct answers in this PR, you should update answers here as well.